### DISCUSSION TITLE
✅ 🔗 Post Filtering (Duplicate)

### DISCUSSION THREAD

**Discussion by heroicP** [elementor/elementor#18938] [Feature Request]
I find myself needing to add the ability to filter posts in a post listing on pretty much every single site I build. I've been using [JetSmartFilters](https://crocoblock.com/plugins/jetsmartfilters/) but this really seams like a feature that should just be included in Elementor. Without any sort of filtering if you have hundreds of posts, with noting more than pagination, the user just has to click through page by page looking for posts they are interested in.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the introduction of the Taxonomy Filter in Elementor Pro v3.15! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
